src/store/consts/static.ts

```diff
@@ -255,3 +255,5 @@ export const reversedAddressTickerMap = Object.fromEntries(
 export const LIQUIDITY_PLOT_DECIMAL = 12n
 
 export const DEFAULT_TOKEN_DECIMAL = 12n
+
+export const MINIMAL_POOL_INIT_PRICE = 0.00000001
```

Comment:
why is it const?

Reply:
We show in the init price input only 8 decimals, even if the lowest tick price is lower. So I have created minimal value to write in input, not to show  0 as init price.